docs/release-notes/RELEASE-NOTES-0.21.0.md

```diff
@@ -7,6 +7,8 @@ It:
 * Fixes miner fee reward calculations, was too generous before. This impacts consensus.
 * Modifies the minimum static component of the fee of oracle transactions to `1` - as for all other transactions. This impacts consensus.
 * Increases beneficiary reward delay to 180 key blocks / generations. This impacts consensus.
+* Adds on-chain forcing of progress for channel off-chain contracts. This
```

Comment:
Copied in https://github.com/aeternity/epoch/wiki/Release-notes-next/_compare/b373e5dfb8f3b15c113b71e88e436393942654c8...df1f035f5162e74e25265af132a96dd3b2d07182